orderer/common/multichannel/registrar_test.go

```diff
@@ -647,6 +647,29 @@ func TestRegistrar_JoinChannel(t *testing.T) {
 	})
 }
 
+func TestRegistrar_RemoveChannel(t *testing.T) {
+	// system channel
+	confSys := genesisconfig.Load(genesisconfig.SampleInsecureSoloProfile, configtest.GetDevConfigDir())
+	genesisBlockSys := encoder.New(confSys).GenesisBlockForChannel("sys-channel")
+
+	cryptoProvider, err := sw.NewDefaultSecurityLevelWithKeystore(sw.NewDummyKeyStore())
+	assert.NoError(t, err)
```

Comment:
There's been a push to standardize using `require` over `assert` in some of the related code paths. I know the rest of this file still mixes both but we may want to standardize starting here.